src/robusta/utils/common.py

```diff
@@ -37,3 +38,11 @@ def duplicate_without_fields(obj: HikaruBase, omitted_fields: List[str]):
             pass  # in case the field doesn't exist on this object
 
     return duplication
+
+
+def parse_query_string(query_string: str) -> Dict[str, str]:
+    if not query_string:
+        return {}
+    query_params = parse_qs(query_string)
+    return {key: value[0] for key, value in query_params.items()}
```

Comment:
It can happen that there is the same key repeated a few times in a query. So value[0] is not always sufficient